VTEX - Catalog API.json

```diff
@@ -20900,6 +20900,16 @@
                             "type": "integer",
                             "default": "1"
                         }
+                    },
+                    {
+                        "name": "onlyAssigned",
+                        "in": "query",
+                        "required": false,
+                        "description": "If set as `true`, it allows user to decide if the SKUs that are not assigned to a specific trade policy should be also returned",
```

Comment:
Actually, if set as `false` the SKUs not assigned to a specific trade policy should be also returned.